tools/devctr/Dockerfile.aarch64

```diff
@@ -53,23 +53,11 @@ RUN apt-get update \
     && python3 -m pip install \
```

Comment:
these need not be in the the requirements file, they just manage the installations, that's why they are special.
They need to be executed before we actually run pip install -r requirements.txt, otherwise the command will break. 
Also even you see they are not part of requirements-freeze which was copied from container. This is because they don't pull any pip vended packages.